Two tests failures in Operon on Arch and Ubuntu 22.04

Steps to reproduce
------------------
Building on Arch:
```
python setup.py build
pytest -v --color=yes -k
```

Expected Output
---------------
Run flawlessly.


Actual Output
-------------
```python
___________________________ TOperonEdit.test_dry_run ___________________________
tests/test_operon.py:368: in test_dry_run
    e = self.check_true(["edit", "--dry-run", self.f], False, True)[1]
tests/test_operon.py:54: in check_true
    return self._check(args, True, stdout_not_empty, stderr_not_empty, **kwargs)
tests/test_operon.py:63: in _check
    self.failUnlessEqual(s == 0, command_succeeds, msg=repr((s, o, e)))
tests/__init__.py:39: in assertEqual
    super().assertEqual(second, first, msg)
E   AssertionError: True != False : (1, '', 'edit: No changes detected\n')
_________________________ TOperonEdit.test_remove_all __________________________
tests/test_operon.py:384: in test_remove_all
    self.check_true(["edit", self.f], False, False)
tests/test_operon.py:54: in check_true
    return self._check(args, True, stdout_not_empty, stderr_not_empty, **kwargs)
tests/test_operon.py:63: in _check
    self.failUnlessEqual(s == 0, command_succeeds, msg=repr((s, o, e)))
tests/__init__.py:39: in assertEqual
    super().assertEqual(second, first, msg)
E   AssertionError: True != False : (1, '', 'edit: No changes detected\n')
```

Test System
-----------

### Which version of Quod Libet?
4.5

### Which operating system 
Arch Linux


Additional Information
----------------------
Version of any dependency used available by clicking on names at https://archlinux.org/packages/community/any/quodlibet/.

--- a/quodlibet/operon/commands.py
+++ b/quodlibet/operon/commands.py
@@ -234,6 +234,7 @@ class EditCommand(Command):
         try:
             try:
                 os.write(fd, dump)
+                os.fsync(fd)
             finally:
                 os.close(fd)
 

